subfun/BIDS_RFX.m

```diff
@@ -243,9 +242,8 @@ function BIDS_RFX(action, mmFunctionalSmoothing, mmConSmoothing, opt)
                     ffxDir = getFFXdir(subNumber, mmFunctionalSmoothing, opt);
 
                     a = Session(j).con;
-                    ConList = dir(sprintf([smoothOrNonSmooth,'con_%0.4d.nii'],con));
-                    range{iGroup}.donnees{iSub,:} = fullfile([ffxDir, ConList.name]);
-
+                    ConList = dir(fullfile(ffxDir,sprintf([smoothOrNonSmooth,'con_%0.4d.nii'],con)));
+                    range{iGroup}.donnees{iSub,:} = fullfile(ffxDir, ConList.name);
```

Comment:
Fixed the path:
fullfile([ffxDir, ConList.name]);  ----->  fullfile(ffxDir, ConList.name);